src/services/notification/notification.channels.ts

```diff
@@ -0,0 +1,26 @@
+import '@feathersjs/transport-commons'
+import { Application, ServiceAddresses } from '../../definitions'
+
+const CHANNEL = 'notifications'
+
+function filterByOwner (app: Application, data: any) {
+  const accounts: string[] = data.accounts
+  return app.channel(CHANNEL)
+    .filter(connection => {
+      const connectedAccount: string = connection.ownerAddress
```

Comment:
Hmm we should rename it from `ownerAddress` at some point, but that has to be coordinated with UI... Lets keep it for now.